projects/network_table_listener/NetworkTableListener.py

```diff
@@ -0,0 +1,239 @@
+from nt_connection.Connection import ConnectionError
+from nt_connection.Connection import Connection
+from nt_connection.Help import Help
+import nt_connection.uri
+import generated_python.Value_pb2 as Value_pb2
+import generated_python.Node_pb2 as Node_pb2
+import websocket
+import json
+import random
+
+
+class NetworkTableListener:
+    def __init__(self, ws_connection):
+        web_socket = websocket.WebSocket()
+        web_socket.connect(ws_connection)
+        self.ws = web_socket
+
+    """
+    The methods below should be used as a callback function when subscribing to the network table.
+    It send all the fields of each sensor in json formatted data to the websocket.
+    ex usage.)
+    - Create a network table instance
+    - Call the method subscribe and input the uri and the callback method for that uri
+    - network_table.subscribe('/wind_sensor_0', getWindSensorData)
+
+    """
+
+    def getWindSensorData(self, node, uri):
+        wind_sensor_data = json.dumps(
+            {
+                'sensor_type': 'wind',
+                'sensor_id': uri,
+                'speed': node.children['iimwv'].children['wind_speed'].value.int_data,
+                'direction': node.children['iimwv'].children['wind_direction'].value.int_data
+                #'current': node.children['uccm'].children['current'].value.int_data,
```

Comment:
I know it's dumb, but can you just add a space after all the '#' - that's what failing the static_analysis

Reply:
done

Reply:
done